whatthelog/syntaxtree/syntax_tree_factory.py

```diff
@@ -57,7 +57,7 @@ def parse_file(self, filepath: str) -> SyntaxTree:
     def __parse(self, configs: dict) -> SyntaxTree:
 
         name = configs["name"]
-        prefix = configs["prefix"]
+        prefix = configs["syntax"]
```

Comment:
This seems to be causing an issue with the tests.

Reply:
Fixed